column.go

```diff
@@ -34,9 +34,22 @@ const (
 	Time ColumnType = "TIME"
 )
 
+// AlterColumnConstraint enum.
+type AlterColumnConstraint uint16
```

Comment:
this constraint is not actual database constraint right? how about`AlterColumnMode`? 
then in the Column definition we can use `AlterMode` field name